pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>strike-off-objections-api</artifactId>
+    <version>unversioned</version>
+    <packaging>jar</packaging>
+
+    <name>strike-off-objections-api</name>
+    <description>Create a new strike-off objection request</description>
+
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-spring-boot-parent</artifactId>
+        <version>1.3.0</version>
+    </parent>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <structured-logging.version>1.2.0-rc1</structured-logging.version>
+        <rest-service-common-library.version>0.0.3</rest-service-common-library.version>
+        <avro.version>1.8.1</avro.version>
+        <ch-kafka.version>1.3.0-rc1</ch-kafka.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+        </dependency>
+        
+        <!-- TODO Include this configuration when data is persisted and queried -->
+
+<!--         <dependency> -->
```

Comment:
Commented these ones out because they mandated adding DB connection properties, which we don't yet know (or need).